### ISSUE TITLE
cannot start a created container using the ID

### ISSUE THREAD

**Issue by cyphar** [kubernetes-incubator/cri-o#111]
Maybe I'm misunderstanding how the interface is meant to work, because I'm expecting it to kinda work like Docker/runC:

```
 % sudo ./ocic pod create --name sandbox0 --config test/testdata/sandbox_config.json
fe6a1bce4f27392b38f7c9066112a82ec39e4220e7ac6635873113168a725e2c
% sudo ./ocic container create --pod fe6a1bce4f27392b38f7c9066112a82ec39e4220e7ac6635873113168a725e2c --config test/testdata/container_config.json
aecc2d844d19f127c0a026e2325a6357e0a59a7c4b60130e63d4e4eab9ab3b0c
% sudo ./ocic container list
ID: default-sandbox0-0-container1-0
Pod: fe6a1bce4f27392b38f7c9066112a82ec39e4220e7ac6635873113168a725e2c
Status: CREATED
Created: 2016-10-07 15:01:02 +1100 AEDT
ID: default-sandbox0-0-infra
Pod: fe6a1bce4f27392b38f7c9066112a82ec39e4220e7ac6635873113168a725e2c
Status: RUNNING
Created: 2016-10-07 15:00:17 +1100 AEDT
% sudo ./ocic container start --id default-sandbox0-0-container1-0
FATA[0000] Starting the container failed: rpc error: code = 2 desc = container with ID starting with default-sandbox0-0-container1-0 not found: ID does not exist
```

Am I using the wrong ID or something? Or is this an actual bug?

**Comment by cyphar** [comment]
Ah, I figured it out. I need to use the actual ID (`aecc2d844d19f127c0a026e2325a6357e0a59a7c4b60130e63d4e4eab9ab3b0c`) in order to make it work. Is there a reason we shouldn't make it possible to use the short name / ID?

**Comment by runcom** [comment]
@cyphar yeah, we want short ids as well to start containers as well.

**Comment by mrunalp** [comment]
@cyphar @runcom The support for short ids for containers did go in couple of days back. Are you on the latest master? So, you should be able to 
```
% sudo ocic ctr start --id aecc
```

**Comment by cyphar** [comment]
@mrunalp I am on the latest master, but the problem was that I was trying to use the **name** of the container. Another solution would be to print the `ID` of the container when doing a `ocic ctr list` -- currently we only show the *name* and the *pod ID*.

**Comment by mrunalp** [comment]
@cyphar Looks like a bug. The ID printed as part of `ocic ctr list` should really be the ID and not the name. I'll fix this today if someones doesn't before I get to it :)

**Comment by runcom** [comment]
> @cyphar Looks like a bug. The ID printed as part of ocic ctr list should really be the ID and not the name. I'll fix this today if someones doesn't before I get to it :)

Fixed it as part of #114
